bench/src/main.rs

```diff
@@ -154,20 +164,32 @@ async fn bench(
     current_slot: Arc<AtomicU64>,
     tx_metric_sx: UnboundedSender<TxMetricData>,
     log_txs: bool,
+    transaction_size: TransactionSize,
 ) -> Metric {
-    let map_of_txs = Arc::new(DashMap::new());
+    let map_of_txs: Arc<DashMap<Signature, TxSendData>> = Arc::new(DashMap::new());
```

Comment:
No need to specify type. May be your ide issue.

Reply:
nah. sometimes is add them if the deduced type is too far away from declaration.